sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.11.1 (Unreleased)
+## 1.12.0 (Unreleased)
 
+### Features
+
+- Add `json` kwarg to `HttpRequest` object. The `json` kwarg takes your data as input and json serializes it for you. Use instead of `data` if you want this behavior. #16695
```

Comment:
Is there a missing word, or do I don't understand the sentence?

Reply:
updated the changelog wording, does the new wording sound better to you?